docs/source/features/version_control.rst

```diff
@@ -2,19 +2,181 @@
  Version Control
 #################
 
-TensorBay supports version control.
-A new version of a dataset can be built upon the previous version.
-:numref:`Figure. %s <version_control>` demonstrates the relations
-between different versions of a dataset.
+TensorBay supports dataset version control.
+There can be multiple versions in one dataset.
 
-.. _version_control:
+********
+ Commit
+********
 
-.. figure:: ../images/version_control.png
-   :scale: 60 %
+The basic element of TensorBay version control system is :ref:`reference/glossary:commit`.
+Each commit of a TensorBay dataset is a **read-only** version.
+Take the `VersionControlDemo Dataset <https://gas.graviti.cn/dataset/data-decorators/VersionControlDemo>`_ as an example.
```

Comment:
new VersionControlDemo Dataset has been uploaded. Please modify the commit id below.
https://gas.graviti.cn/dataset/graviti-open-dataset/VersionControlDemo/branch/main